bundle/manifests/ingressnodefirewall.openshift.io_ingressnodefirewalls.yaml

```diff
@@ -182,6 +182,27 @@ spec:
                             required:
                             - protocol
                             type: object
+                            x-kubernetes-validations:
```

Comment:
was this auto generated by kube builder?

Reply:
yes all manifests and bundle are results of `make manifests` and `make bundle`